### PR TITLE
Redirect docs to current version

### PR DESCRIPTION
**Pull request by glarizza**
PROBLEM:

See #350 for context. There's a GitHub issue open for this on the
facebook docusaurus repo:
https://github.com/facebook/docusaurus/issues/9049

SOLUTION:

Use the redirect workaround for the time being.

OUTCOME:

https://holos.run/docs will link to the most recent version of our docs
site.

### CODE REVIEW

File: doc/website/static/_redirects [hunk 1]
```diff
@@ -0,0 +1 @@
+/docs /docs/v1alpha5 301
```
Comment (jeffmccune) [inline doc/website/static/_redirects:1]:
```suggestion
/docs /docs/v1alpha5/ 301
/docs/ /docs/v1alpha5/ 301
```

**Comment by natemccurdy** [comment]
Just so I understand this correctly, the `_redirect` file is service-side redirect config supported by CloudFlare, and we will have to update the redirect any time a new API version is pushed (e.g. v1alpha6)?

**Comment by jeffmccune** [comment]
> the _redirect file is service-side redirect config supported by CloudFlare, and we will have to update the redirect any time a new API version is pushed (e.g. v1alpha6)?

@natemccurdy Yes that's correct.

```shell
curl https://gl-350-docs-redirect.holos.pages.dev/docs
```
```txt
Redirecting to /docs/v1alpha5
```

**Comment by jeffmccune** [comment]
FYI, @glarizza @natemccurdy Cloudflare tacks a / on the end of all the URL's, so I changed the redirects to match and handle both `/docs` and `/docs/`

**Pull request merged**